Source/CarthageKit/Xcode.swift

```diff
@@ -1017,12 +1034,62 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 }
 
+
+public func getSecuritySigningIdentities() -> SignalProducer<String, CarthageError> {
+	let securityTask = TaskDescription(launchPath: "/usr/bin/security", arguments: ["find-identity", "-v", "-p", "codesigning"])
```

Comment:
Could you please add spaces as follows: `[ "find-identity", "-v", "-p", "codesigning" ]`